### PR TITLE
Cross functional team slice

### PR DESCRIPTION
**Pull request by matt-paul**
### Motivation

- [Trello ticket](https://trello.com/c/6YaFLvZB/97-what-we-do-cross-functional-slice)

- [Zeplin design](https://app.zeplin.io/project/57c8297fc3cf63795b6a4520/dashboard)

TODO: write a motivation.

### Test plan

Note: 
 - Heading is larger than design as per instruction, in order to march headings elsewhere in the site (24px => 30px)
 - Only the 'Tech' box is a link at present, so the the coloured link arrows have been removed from the other boxes

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by dpiatek** [comment]
hey @matt-paul could you add some screenshots of this?

**Comment by matt-paul** [comment]
screen shot 2018-03-12 at 14 11 54
screen shot 2018-03-12 at 14 05 49
screen shot 2018-03-12 at 14 04 36

File: site/components/colour-box/style.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+.description {
+  composes: lightSerif from "../../css/typography/_fonts.css";
+  color: #F8F8F8;
+  font-size: 20px;
+  line-height: 1.2em;
+}
+
+a {
+  position: relative;
```
Comment (dpiatek) [inline site/components/colour-box/style.css]: Why is this style needed?

File: site/components/colour-box/style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+a .description:hover {
+  border-bottom: 2px solid badgerRed;
+}
```
Comment (dpiatek) [inline site/components/colour-box/style.css]: Why does this need an `a` in front of it?

File: site/components/image/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <img src={src} alt={alt || ''} role={role} className={className} />
+    </picture>
+  );
+};
+
+Image.defaultProps = {
+  alt: '',
+  className: undefined,
+  role: undefined,
+  src: undefined,
```
Comment (dpiatek) [inline site/components/image/index.js]: I don't think we are doing the right thing here. you are providing `alt` as default but then also have an `||` in the code. You are providing `undefined` as defaults but these will be displayed as `undefined` in the HTML. Maybe you are misunderstanding how optional parameters work in flow https://flow.org/en/docs/types/functions/#toc-optional-parameters ?

File: site/pages/what-we-do/cross-functional-slice/index.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    description: 'Together we build a capability and lasting change.',
+  },
+];
+
+const CrossFunctionalSlice = () => (
+  <section className={styles.crossFunctionalSlice}>
+    <div className={styles.photoContainer}>
+      <Image
+        src={crossFunctionalBadgers}
+        alt="working in a cross functional team"
```
Comment (dpiatek) [inline site/pages/what-we-do/cross-functional-slice/index.js]: Think we should improve this alt text to be more descriptive

File: site/pages/what-we-do/cross-functional-slice/styles.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    margin: 42px 0 30px;
+  }
+
+  .imageContainer {
+    padding: 0;
+  }
+  
+  .colourBoxContainer {
+    justify-content: center;
+    /* padding: 40px 0 0 60px; */
```
Comment (dpiatek) [inline site/pages/what-we-do/cross-functional-slice/styles.css]: Can we remove this?

File: site/pages/what-we-do/cross-functional-slice/styles.css [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    max-width: 820px;
+  }
+
+  .boxWrapper {
+    padding: 0 20px 30px;
+    width: 50%;
+    max-width: calc(280px + 40px);
+  }
+
+
```
Comment (dpiatek) [inline site/pages/what-we-do/cross-functional-slice/styles.css]: 2 extra new lines here

File: site/components/colour-box/style.css (hunk 1 above)
Comment (matt-paul) [reply site/components/colour-box/style.css]: removed

File: site/components/colour-box/style.css (hunk 2 above)
Comment (matt-paul) [reply site/components/colour-box/style.css]: @dpiatek At this stage, only one of these colour boxes is a link, with the rest planned to become links over time.  We only want the hover behaviour on the links.

File: site/pages/what-we-do/cross-functional-slice/styles.css (hunk 5 above)
Comment (matt-paul) [reply site/pages/what-we-do/cross-functional-slice/styles.css]: yep!

File: site/pages/what-we-do/cross-functional-slice/styles.css (hunk 6 above)
Comment (matt-paul) [reply site/pages/what-we-do/cross-functional-slice/styles.css]: removed

File: site/pages/what-we-do/cross-functional-slice/index.js (hunk 4 above)
Comment (matt-paul) [reply site/pages/what-we-do/cross-functional-slice/index.js]: I have changed this to 'a photo showing Red Badger employees working in a cross functional team'

File: site/components/image/index.js (hunk 3 above)
Comment (matt-paul) [reply site/components/image/index.js]: updated

File: site/components/colour-box/style.css (hunk 2 above)
Comment (dpiatek) [reply site/components/colour-box/style.css]: that's fine but then you don't need the `a` in that case right? The hover happens on whatever the `.description` is?

Comment (matt-paul) [reply site/components/colour-box/style.css]: no, the hover should only happen if the description is within the link.  Three elements have the class 'description' but should not have the hover behaviour as the element is not within a link

Comment (dpiatek) [reply site/components/colour-box/style.css]: In that case it should have it's own class ... by doing element selectors in css modules, like the one above, we essentially defeat the purpose of CSS modules

Comment (matt-paul) [reply site/components/colour-box/style.css]: ok cool, updated!

**Pull request merged**